installtorqapp.sh

```diff
@@ -0,0 +1,259 @@
+#!/bin/bash
+#Example usage:
+#sh installtorqapp.sh torq=TorQ-3.7.0.tar.gz releasedir=deploy data=datatemp installfile=TorQ-Finance-Starter-Pack-1.9.0.tar.gz
```

Comment:
is there a reason this isn't something like

```suggestion
#sh installtorqapp.sh -torq TorQ-3.7.0.tar.gz -releasedir deploy -data datatemp -installfile TorQ-Finance-Starter-Pack-1.9.0.tar.gz
```

seems like a more common way of passing args to a bash script?

Reply:
From what I found on command line parameters in bash, then the - flags only work properly when they are single characters like -t  TorQ-3.7.0.tar.gz -r deploy and so on. 
Making them long command line parameters I did not find an elegant option that would have the functionality that it has currently. 
Can use getopt, but at the same time having as simple output line with the same functionality I did not find possible. 
Did you have anything else other than getopt in mind?

Reply:
I think there are a couple of options - you could take a look at how flags are handled in `torq.sh` (I think it's good if we can match the argument patterns as closely as possible between the two scripts). For some other examples of implementations, this SO answer seems pretty decent: https://stackoverflow.com/a/14203146/8694303

Reply:
Talking with Jonny the behaviour will be as follows:
bash installtorqapp.sh --torq TorQ-3.7.0.tar.gz --releasedir deploy --data datatemp --installfile TorQ-Finance-Starter-Pack-1.9.0.tar.gz

As torq.sh command line parameters that have -csv for example isn't straightforward to implement and to retain the free ordering of variables. 

This behaviour still retains the free ordering of the parameters and is more alike with what torq.sh has and kdb has.